### ISSUE TITLE
Rename internal Log class

### ISSUE THREAD

**Issue by IliaEremin** [pardom-zz/ActiveAndroid#471]
This is confusing to have android build-in `Log` and active android `Log` classes.

**Comment by azizimusa** [comment]
i agree. I too have many times mistakenly using activeandroid Log for displaying output in logcat.
